### DISCUSSION TITLE
Corsair One Pumps

### DISCUSSION THREAD

**Discussion by npapatheodorou** [liquidctl/liquidctl#468] [Q&A]
I recently got a Corsair one and I was wondering if it's possible to configure its pumps with liquidctl. Of course it is not listed and I haven't seen any Corsair one supported. Is there a particular reason why (hardware/kernel limitations) ? If not, from where should I start to make it work. Is there any tutorial or something so I can try add my "own" device ?

**Comment by jonasmalacofilho** [comment]
Duplicate of #467.
